docs_src/index.html.md

```diff
@@ -0,0 +1,469 @@
+
+# Deploying Your Project
+
+When deploying a project, you typically deploy to a "remote".
+
+A "remote" is a remote VPS, such as a [Google Cloud Compute](https://cloud.google.com/compute/)
+or [AWS Elastic Cloud Compute (EC2)](https://aws.amazon.com/ec2/) instance. These
+are computers in the "cloud" that will be used to deploy your project, so that
+you don't have to use your own.
+
+If this is your first time setting up a VPS, jump ahead to the
+[Provisioning a Remote](#provisioning-a-remote) section, which will help you set
+up a VPS for Inertia.
+
+## Using an Existing Remote
+
+> This command will prompt you for the path to your PEM file, your username,
+> and the IP address of your remote. These parameters will be used to execute
+> SSH commands that set up Inertia on your VPS.
+
+```shell
+inertia remote add ${remote_name}
+```
+
+To use an existing remote, you'll need its address and a PEM key that can be
+used to access it. Inertia will also need a few ports exposed, namely one for
+the Inertia daemon (`4303` by default) and whatever ports you need for your
+deployed project.
+
+<aside class="notice">
+If you use a non-standard SSH port (ie not port 22) or want to use a different
+port for the Inertia daemon, use the <code>--ssh.port ${port}</code> and
+<code>--port ${port}</code> flags respectively.
+</aside>
+
+## Provisioning a Remote
+
+```shell
+inertia provision ${cloud_provider} ${remote_name}
+```
+
+Inertia has integrations with some cloud providers to allow you to easily
+provision a new VPS instance and set it up for Inertia. You can run `inertia
+provision --help` to see what options are available.
+
+### Example: Provisioning an EC2 Instance
+
+```shell
+inertia provision ec2 my_remote \
+  --from-profile \
+  --ports 8080
+```
+
+> This command says: "provision an ec2 instance called 'my_remote' using
+> credentials from my AWS profile and expose port 8080 for my project".
+
+TODO: IAM setup, saving profile in aws config, choosing an image etc.
+
+## Deployment Configuration
+
+> An example `inertia.toml`:
+
+```toml
+# ... other stuff
+
+[remotes]
+  [remotes.my_remote]
+    IP = "ec2-203-0-113-25.compute-1.amazonaws.com"
+    user = "root"
+    pemfile = "/Users/robertlin/.ssh/id_rsa"
+    branch = "master"
+    ssh-port = "22"
+    [remotes.my_remote.daemon]
+      port = "4303"
+      token = ""
+      webhook-secret = "abcdefg"
+```
+
+> To change a setting, you can edit the configuration file directly, or run:
+
+```shell
+inertia remote set ${remote_name} ${parameter} ${new_value}
+```
+
+> For example, the following will change the `user` associated with `my_remote`
+> to `bobheadxi` and print out the new settings:
+
+```shell
+inertia remote set my_remote user bobheadxi
+inertia remote show my_remote
+```
+
+Once you've added a remote, remote-specific settings are available under the
```

Comment:
You might want to clarify that this should all be correctly filled out by `provision`.

Reply:
good point - actually, this entire section can be skipped if you use `provision`. I've added a little notice thing 👍